gitmanager.py

```diff
@@ -133,13 +122,6 @@ def add_to_blacklist(cls, **kwargs):
                     blacklist_file.write("\n")
                 blacklist_file.write(item_to_blacklist + "\n")
 
-            # Checkout a new branch (for PRs for non-code-privileged people)
-            branch = "auto-blacklist-{0}".format(str(time.time()))
-            git.checkout("-b", branch)
-
-            # Clear HEAD just in case
-            git.reset("HEAD")
```

Comment:
This should be kept *somewhere*.  We need to make sure we're actually on `HEAD` and not in a detached or reverted state.  Which is incredibly important.

Reply:
In your test case in CHQ room, you detailed why this is important - you clobber remote commits that may be just as important.  Therein lies the core problem and why we left this here.

Reply:
Discussion: https://chat.stackexchange.com/rooms/11540/conversation/pr-1216